tests/tutorials/test_tutorials.py

```diff
@@ -114,6 +114,9 @@ def test_gluon_save_load_params():
 
 def test_gluon_hybrid():
     assert _test_tutorial_nb('gluon/hybrid')
+
+def test_gluon_hybrid():
```

Comment:
Thanks for your contribution! It seems to be test_gluon_performance.

Reply:
Great catch @wkcn!

Reply:
Updated.